k8s/upgrade-job/Cargo.toml

```diff
@@ -23,3 +23,4 @@ serde_json = "1.0.93"
 serde_yaml = "0.9.17"
 serde = { version = "1.0.152", features = ["derive"] }
 semver = { version="1.0.17", features = ["serde"] }
+const_format = "0.2.30"
```

Comment:
nit: I'm tempted to say just use normal format, we don't have any high performance runtime constraints here, so it's better to reduce compile times.

Reply:
Blazingly fast 😎, with those two literals.
Okay, I'll get rid of them.